actors/paych/tests/paych_actor_test.rs

```diff
@@ -773,15 +773,14 @@ mod update_channel_state_extra {
         rt.expect_send(
```

Comment:
There is a bug in paych that makes builtin actors consensus break from specs-actors.  To reproduce it this test needs a little cleanup.  First set sv.extra.data to `fake_params` not the cbor serialization of fake_params.  Then expect params equal to fake_params in this expect_send call.

(I think the use of cbg.Deferred/builtin.CBORBytes in go introduced the confusion, these types treat raw bytes as cbor without wrapping them in a cbor byte string)

Reply:
@lemmih the fix in paych code should be to remove the serialize around &extra.data in the `send` call.

Reply:
Not strictly required but you can back port this change to v7 actors too so a separate PR with the fix might help

Reply:
Is it fixed now?

Reply:
Yeah looks good